manager.go

```diff
@@ -39,12 +39,10 @@ type Manager struct {
 	connMu      sync.RWMutex
 	conn        rpc.ClientConn
 	client      pb.AgentDeviceServiceClient
-	partID      string
-	cloudAddr   string
-	cloudSecret string
+	cloudConfig *logging.CloudConfig
 
 	logger      logging.Logger
-	netAppender *logging.NetAppender
+	NetAppender *logging.NetAppender
```

Comment:
I'm a bit lost here... First, why make this public? Second, I can't find where this is set now, since you removed the "attachNetAppender()" function.